_data/toc.yaml

```diff
@@ -917,72 +917,20 @@ manuals:
-        title: Push the image to Docker Cloud's Registry
-      - path: /docker-cloud/getting-started/deploy-app/5_deploy_the_app_as_a_service/
-        title: Deploy the app as a Docker Cloud service
-      - path: /docker-cloud/getting-started/deploy-app/6_define_environment_variables/
-        title: Define environment variables
-      - path: /docker-cloud/getting-started/deploy-app/7_scale_the_service/
-        title: Scale the service
-      - path: /docker-cloud/getting-started/deploy-app/8_view_logs/
-        title: View service logs
-      - path: /docker-cloud/getting-started/deploy-app/9_load-balance_the_service/
-        title: Load-balance the service
-      - path: /docker-cloud/getting-started/deploy-app/10_provision_a_data_backend_for_your_service/
-        title: Provision a data backend for the service
-      - path: /docker-cloud/getting-started/deploy-app/11_service_stacks/
-        title: Stackfiles for your service
-      - path: /docker-cloud/getting-started/deploy-app/12_data_management_with_volumes/
-        title: Data management with Volumes
-  - sectiontitle: Manage Applications
-    section:
-    - path: /docker-cloud/apps/deploy-to-cloud-btn/
-      title: Add a Deploy to Docker Cloud button
-    - path: /docker-cloud/apps/auto-destroy/
-      title: Automatic container destroy
-    - path: /docker-cloud/apps/autorestart/
-      title: Automatic container restart
-    - path: /docker-cloud/apps/auto-redeploy/
-      title: Automatic service redeploy
-    - path: /docker-cloud/apps/load-balance-hello-world/
-      title: Create a proxy or load balancer
-    - path: /docker-cloud/apps/deploy-tags/
-      title: Deployment tags
-    - path: /docker-cloud/apps/stacks/
-      title: Manage service stacks
-    - path: /docker-cloud/apps/ports/
-      title: Publish and expose service or container ports
-    - path: /docker-cloud/apps/service-redeploy/
-      title: Redeploy running services
-    - path: /docker-cloud/apps/service-scaling/
-      title: Scale your service
-    - path: /docker-cloud/apps/api-roles/
-      title: Service API Roles
-    - path: /docker-cloud/apps/service-links/
-      title: Service discovery and links
-    - path: /docker-cloud/apps/triggers/
-      title: Use triggers
-    - path: /docker-cloud/apps/volumes/
-      title: Work with data volumes
+    - path: /docker-cloud/cloud-swarm/
+      title: Swarms in Docker Cloud
+    - path: /docker-cloud/cloud-swarm/using-swarm-mode/
+      title: Using Swarm Mode
+    - path: /docker-cloud/cloud-swarm/register-swarms/
+      title: Register existing swarms
+    - path: /docker-cloud/cloud-swarm/create-cloud-swarm/
+      title: Create a new swarm in Docker Cloud
+    - path: /docker-cloud/cloud-swarm/connect-to-swarm/
+      title: Connect to a swarm through Docker Cloud
+    - path: /docker-cloud/cloud-swarm/link-aws-swarm/
+      title: Link to Amazon Web Services to create Swarms
   - sectiontitle: Manage Builds and Images
```

Comment:
Elevate the "Manage Build and Images" section to appear above the "Swarm Mode (Beta)" section.